### DISCUSSION TITLE
Why no new features? Is there a replacement crate?

### DISCUSSION THREAD

**Discussion by damccull** [zip-rs/zip-old#305] [Archive]
The readme says this crate will get only bug fixes and no new features. Why's that? Is there a replacement or newer crate somewhere?

**Comment by zamazan4ik** [comment]
@Plecra since this question is appearing again, I suggest you place somewhere explanation (I suggest in README).

@damccull regarding your question I will copypaste here the answer from @Plecra (the library owner) from our Discord:

> Heya, sorry about that, I'll make sure to make it clearer
> I am meaning to refer to the 0.6 version - we are specifically prioritising fixes to the library as it is due to the internal complexity making reviewing PRs slow. This is being improved with an upcoming release (decompression & decryption support already supported, writing zip files is working and needs a few fixes) so that we can sustain looking after the crate.
> It definitely isn't dying, and I'm actually more actively maintaining than I have been for quite a while. Zamazan is really helping with being able to focus on moving the crate forward

**Reply by damccull** [reply]
Ah! I see. That's great news! I would also suggest putting that text or very similar in the RANGE where it currently says that it's only getting fixes. The current language makes it seem like the project is barely on life support.

**Comment by Plecra** [comment]
The notice is gone, problem solved! ;)

**Reply by damccull** [reply]
Totally makes sense. Thanks.
